### PR TITLE
test: Extended the feature test framework

### PR DESCRIPTION
**Pull request by energinet-ajw**
The framework can now handle four tables from the wholesale results internal database.
A test example has been added.

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/__init__.py
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/__init__.py:1]: Remove?

File: source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/test_output.py
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/test_output.py:1]: Remove?

File: source/databricks/calculation_engine/package/databases/wholesale_results/wholesale_results_internal_repository.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+class WholesaleResultsInternalRepository:
+    def __init__(
+        self,
+        spark: SparkSession,
+        catalog_name: str,
+    ) -> None:
+        self._spark = spark
+        self._catalog_name = catalog_name
+
+    def read_energy_results(self) -> DataFrame:
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/package/databases/wholesale_results/wholesale_results_internal_repository.py:37]: Is the "_results" suffix needed in each function name?

File: source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Energinet DataHub A/S
```
Comment (BjarkeMeier) [inline source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/__init__.py:1]: FYI: Copyright headers are not required.

File: source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/__init__.py
Comment (energinet-ajw) [reply source/databricks/calculation_engine/tests/features/energy_calculation/given_temp_ajw/__init__.py:1]: It is an example for Claus to use. I will ask him about it.

**Pull request merged**